src/components/styles.css

```diff
@@ -7,27 +7,26 @@
   font-family: 'Gill Sans', 'Gill Sans MT', 'Trebuchet MS', sans-serif;
 }
 
+body {
+  width: 100%;
+  background-color: #f5f6fa;
+}
+
 .nav-bar {
   display: flex;
   flex-direction: row;
   width: 100%;
-  background: #c73e1d;
+  background-color: #fff;
   height: 4.938rem;
 }
```

Comment:
- [ ] Kindly justify the contents of the navbar to the center so your header style matches the original design just as the previous reviewer highlighted.

Hint:
Add this style to this nav-bar class `justify-content: center;` and things will be perfect as required 👍 

**Your Styles**
![image](https://user-images.githubusercontent.com/50941074/191872052-827bee91-ac2d-4fda-9fe7-9682fc99c869.png)

**The original design**
![image](https://user-images.githubusercontent.com/50941074/191872116-ec9adc76-2631-41f1-ae99-c3522b5bc679.png)